### DISCUSSION TITLE
hello world failed

### DISCUSSION THREAD

**Discussion by HeathRelease** [emilk/egui#5479] [Q&A] [answered]
I download this repo, and open it as rust project by rustrover, then run examples/hello_world, the ERROR hapend.
![1734329742770](https://github.com/user-attachments/assets/c182faeb-b7bf-47ad-b3fa-f5a850a5f4e3)
file exist
![image](https://github.com/user-attachments/assets/6404404f-9ae5-4982-99e0-4b25bbc5d597)

**Comment by lucasmerlin** [comment]
You'll need to [setup git lfs](https://github.com/emilk/egui/blob/master/CONTRIBUTING.md#making-a-pr)

**Reply by YgorSouza** [reply]
This should have been fixed by #5480, right? So now you only need LFS if you actually want to contribute to egui, not just run the examples.

**Reply by lucasmerlin** [reply]
Yeah 👍

**Reply by HeathRelease** [reply]
I tried git lfs and it works, but I downloaded the images from another example:images from my browser in jpg format, I replaced the path and the loading fails again!
Download path(at example/images,line 32):
```rs
ui.add(
                    egui::Image::new("https://picsum.photos/seed/1.759706314/1024").rounding(10.0),
                );
```
load that .jpg file:
```rs
ui.image(egui::include_image!(
                "1062.jpg"
            ));
```
![image](https://github.com/user-attachments/assets/d2029ac5-bf06-4c3a-93eb-2a2008c2feef)
PNG is ok
![image](https://github.com/user-attachments/assets/6435f6a0-bd0e-4e5a-85f3-6b6380373120)

**Reply by lucasmerlin** [reply, marked as answer]
You need to enable the jpeg feature of the image crate
